presto-docs/src/main/sphinx/release/release-0.217.rst

```diff
@@ -0,0 +1,60 @@
+=============
+Release 0.217
+=============
+
+General Changes
+---------------
+
+* Fix poor query planning performance for queries submitted when the cluster does not have the minimum required number of workers available.
+* Fix a bug that caused a union of a table with a ``VALUES`` statement to execute on a single machine, which may result in out of memory errors.
+* Improve CPU usage by removing unnecessary remote exchanges from the logical plan before window functions if the data is already available locally.
+* Add grouped execution support for window functions.
+* Add support for the ``BOOLEAN`` type for ``EXPLAIN IO`` statements.
+* Add support for choosing distribution type for semi joins based on estimated cost when the session property ``join_distribution_type='AUTOMATIC'`` is set.
+* Add support for collecting table statistics on demand with the ``ANALYZE`` statement.
+* Add a config option (``query.stage-count-warning-threshold``) to specify a per-query threshold for the number of stages.
```

Comment:
We use both `config option` and `configuration property` in this doc, let's be consistent and always use `configuration property`.